ignite3/src/main/java/site/ycsb/db/ignite3/IgniteSqlClient.java

```diff
@@ -27,6 +28,24 @@ public class IgniteSqlClient extends AbstractSqlClient {
 
   private static final AtomicInteger SQL_INIT_COUNT = new AtomicInteger(0);
 
+  /** Statement for reading values. */
+  private static final ThreadLocal<Statement> READ_STATEMENT = ThreadLocal
+      .withInitial(IgniteSqlClient::buildReadStatement);
```

Comment:
We should also close statement instances when they're created. Looks like it should be done in `cleanup()` method.